From f0453768842a463c30676570720527fe1bf71e58 Mon Sep 17 00:00:00 2001
From: Adarsh-MR <adarshmr1998@gmail.com>
Date: Sun, 15 Oct 2017 11:33:28 +0000
Subject: [PATCH 2/4] Themes: Settings: Unlink Network & Internet and Wifi
 dashboard icon

* Currently its using Wifi icon for both "Wifi" and "Network & internet" Dashboard icon
* This will allow themers to provide diff icon for Network & Internet and  Wifi
* Thanks to Gurpreet (SNN Infinity) for headsup on using layer-list

Change-Id: I50bb1789792c363788908b32adcfb1c5f482a465
Signed-off-by: Adarsh-MR <adarshmr1998@gmail.com>
---
 AndroidManifest.xml                  |  2 +-
 res/drawable/ic_settings_network.xml | 19 +++++++++++++++++++
 2 files changed, 20 insertions(+), 1 deletion(-)
 create mode 100644 res/drawable/ic_settings_network.xml

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 5622d56..06d78c8 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -148,7 +148,7 @@
         <activity android:name=".Settings$NetworkDashboardActivity"
             android:taskAffinity="com.android.settings"
             android:label="@string/network_dashboard_title"
-            android:icon="@drawable/ic_settings_wireless"
+            android:icon="@drawable/ic_settings_network"
             android:parentActivityName="Settings">
             <intent-filter android:priority="1">
                 <action android:name="android.settings.WIRELESS_SETTINGS" />
diff --git a/res/drawable/ic_settings_network.xml b/res/drawable/ic_settings_network.xml
new file mode 100644
index 0000000..fdd5da2
--- /dev/null
+++ b/res/drawable/ic_settings_network.xml
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) 2017 Projekt Substratum
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:drawable="@drawable/ic_settings_wireless" />
+</layer-list>
-- 
2.1.2

